tomtom-navigation-core-examples/usecase/src/main/kotlin/com/example/usecase/MainActivity.kt

```diff
@@ -415,8 +415,9 @@ class MainActivity : AppCompatActivity() {
      * Set the bottom padding on the map. The padding sets a safe area of the MapView in which user interaction is not received. It is used to uncover the chevron in the navigation panel.
      */
     private fun setMapNavigationPadding() {
-        val paddingBottom = resources.getDimensionPixelOffset(R.dimen.map_padding_bottom)
```

Comment:
Hi @AristotelispetrosBobas-TomTom, can you have a look at this, I think it is a potential error because the chevron doesn't appear on the screen. Can you double check that?